Mono.TextTemplating.Tests/GenerationTests.cs

```
@@ -59,6 +59,16 @@ public void ImportReferencesTest ()
 			Assert.IsNull (gen.Errors.OfType<CompilerError> ().FirstOrDefault (), "ProcessTemplate");
 		}
 
+		[Test]
+		public void IncludeFileThatDoesNotExistTest ()
+		{
+			var gen = new TemplateGenerator ();
+			string tmp = null;
+			gen.ProcessTemplate (null, "<#@ include file=\"none.tt\" #>", ref tmp, out tmp);
+			Assert.IsTrue (gen.Errors.OfType<CompilerError> ().FirstOrDefault ().ErrorText
```

This should probably be `First()` rather than `FirstOrDefault()` so that the error when it fails is "no objects" rather than "null reference".

---

I was just keeping it consist with the test above.  I can change it if you want me to.